config/routes/budget.rb

```diff
@@ -1,6 +1,6 @@
 resources :budgets, only: [:show, :index] do
   resources :groups, controller: "budgets/groups", only: [:show]
-  resources :investments, controller: "budgets/investments", only: [:index, :new, :create, :show, :destroy] do
+  resources :investments, controller: "budgets/investments", only: [:index, :new, :create, :show, :destroy, :edit, :update] do
```

Comment:
Since now we're using all seven default controller actions, we could probably remove the `only` key and write: `resources :investments, controller: "budgets/investments" do`